lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Tab = props => {
-  /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
-      if they match, the className should be: 'tab active-tab', 
-      if it is not it should just be 'tab'*/
+if (props.tab)
   return (
     <div
-      className={''}
-      onClick={() => {
-        /* Replace this dummy click handler function with your selectTabHandler function from props 
-         you'll need to pass the `tab` in as an argument to this handler. */
-      }}
-    >
+    className={
+      props.selected === props.selectedTab ? 'tab active-tab' : 'tab'
+    }
+      onClick={() => props.selectTabHandler(props.tab)}>
       {props.tab.toUpperCase()}
     </div>
   );
-};
+}
 
-// Make sure you include PropTypes on your props.
 
-export default Tab;
+Tab.PropTypes = {
```

Comment:
should be `Tab.propTypes`. Remember to check for errors and warnings in the console